server/athenian/api/openapi/openapi.yaml

```
@@ -1431,7 +1431,7 @@ components:
       - review
       - merge
       - release
-      - done  # PR afterlife - PRs that were released and nothing ever happens to them
+      - done  # PR afterlife - PRs that were released or closed and nothing ever happens to them
```

Do you mean rejected?
```suggestion
      - done  # PR afterlife - PRs that were released or rejected and nothing ever happens to them
```

---

Much more clear, thanks